src/parse.cpp

```diff
@@ -1,7 +1,20 @@
 #include <iostream>
+#include <fstream>
+#include <sstream>
+
 #include "parse.hpp"
+#include "lexer.h"
 
 int parse() {
-    std::cout << "Parsing" << std::endl;
+    // Passing in the program file & modifiying it 
+    std::ifstream inFile;
+    inFile.open("test2.txt"); // Keep in mind program file directory ! ! !
```

Comment:
Now that the CLI can be used to specify a file, there's no need to hardcode the filename.